src/solver/optimisation/opt_construction_matrice_des_contraintes_cas_lineaire.cpp

```diff
@@ -1238,13 +1234,11 @@ void OPT_ConstruireLaMatriceDesContraintesDuProblemeLineaire(PROBLEME_HEBDO* Pro
     }
 
     // Export structure
-    if (exportStructure)
+    if (doWeExport)
     {
-        OPT_ExportVariables(*study, varname, "variables", "txt", numSpace);
-
-        // TODO : for now empty constraints.txt file needed
```

Comment:
Export file **constraints.txt** is empty and useless, so I removed it.
If it turns out to be useful at some point, we will add it easily.